addons/get_info/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "UDES Get Info",
```

Comment:
Sorry to be awkward, but can we rename this module to `udes_get_info`? I know it wasn't done like that on the v13 module but it should have been. Anything in `udes-open` repo should be prefixed with `udes_`.